go-controller/pkg/ovn/gateway_init.go

```diff
@@ -115,17 +115,24 @@ func (oc *Controller) gatewayInit(nodeName string, clusterIPSubnet []*net.IPNet,
 		gwLRPNetworks = append(gwLRPNetworks, gwLRPIfAddr.String())
 	}
 
+	var options map[string]string
+	if !config.Gateway.DisableGatewayMTU {
+		options = map[string]string{"gateway_mtu": fmt.Sprintf("%d", util.GetMaxFrameLength())}
```

Comment:
[A new feature was added](https://github.com/ovn-org/ovn/commit/acffcc0dde6c641aeaf516e0b600813c7bc4d489) to upstream OVN recently, `gateway_mtu_bypass`, we should probably use it to bypass check_pkt_larger checks for types of traffic that don't need it, for example for TCP.

Reply:
@trozet Do you think this new feature (gateway_mtu_bypass) would address partially [your concern](https://github.com/ovn-org/ovn-kubernetes/pull/2490#issuecomment-953120465) of no rate limiting for ovn-controller?